### ISSUE TITLE
[WANTED] MAINTAINERS

### ISSUE THREAD

**Issue by rijnhard** [pkgcloud/pkgcloud#617]
For those who haven't noticed pkgcloud has not been actively maintained for a while (which I know has affected my usage of it), so after tweeting at @indexzero and @rosskukulinski they are willing to support another maintainer.

So anyone who is willing to get involved please make yourself heard here.

CC
@mattgrande 

_Edit:_ and Post Timezones

Mine is GMT+2 (SAST/CAT)

**Comment by mattgrande** [comment]
Yep, I'm interested.

**Comment by rosskukulinski** [comment]
Thanks for opening this issue @rijnhard.  Please also post your timezones so I can schedule a welcome call for interested folks next week (Week of July 23rd).

**Comment by indexzero** [comment]
Can you include me and @jcrugzz on that as well @rosskukulinski?

**Comment by papa-rugi** [comment]
Looking to get involved as well! @rosskukulinski  @rijnhard

**Comment by bhanu2507** [comment]
I'm interested.

**Comment by rosskukulinski** [comment]
Scheduled a call for Wednesday July 25th @ 11am ET -- trying to balance west coast & EU is always hard.

If your GitHub profile has a publicly listed email address, you should have a calendar invite in your inbox. If you'd like one, please email me ross at heptio.com and I'll add you to the invite.

Join from PC, Mac, Linux, iOS or Android: https://zoom.us/j/724897890

Or iPhone one-tap :
    US: +16465588656,,724897890# or +16699006833,,724897890# 
Or Telephone:
    Dial(for higher quality, dial a number based on your current location): 
        US: +1 646 558 8656 or +1 669 900 6833 
    Meeting ID: 724 897 890
    International numbers available: https://zoom.us/u/cvsvYMQn


<img width="718" alt="pkgcloud" src="https://user-images.githubusercontent.com/2746479/43022782-63b39f60-8c36-11e8-995e-cb997b693fd0.png">

**Comment by rosskukulinski** [comment]
I waited 15 minutes and nobody showed up. ¯\_(ツ)_/¯

**Comment by papa-rugi** [comment]
Oh my gosh, I am SO sorry. I totally spaced on the time zone thing(I’m on
west coast) would you be able to meet in a few hours?

**Comment by papa-rugi** [comment]
Is there some way we can make this work? I’m really passionate about
working on this project. Would love to meet again, let me know!! So sorry
for standing you up.

**Comment by papa-rugi** [comment]
Hey Ross. SO sorry for standing you up. I got the time zones totally wrong.
If there is any way we can meet again, this is a project that I’m really
passionate about working on.

Thanks,
Alexander

**Comment by rosskukulinski** [comment]
Hi @papa-rugi - I suggest you touch base with @mattgrande who is now a pkgcloud contributor.  I'm hopeful he will start to get the project organized moving forward.  

One area that we sorely need help on is reworking the automated tests.  Some of the cloud providers in the test suite no-longer exist -- they should be removed.  Many of the test mocks are probably out-of-date with newer cloud packages.  Maybe you could start to take a look at some of those items?

**Comment by rosskukulinski** [comment]
@mattgrande / @papa-rugi done

**Comment by dantman** [comment]
How is it going with maintenance?

At minimum we need someone who can just merge PRs and make releases.

I don't mind reviewing and submitting PRs for issues that affect me.

**Comment by rudimk** [comment]
I'd really really like to contribute, especially by starting off with support for GCP compute and object storage. 

I think I'll go ahead, dive into the APIs being exposed over the next couple of days, and hammer out something for GCE.

**Comment by walkerandco** [comment]
Forgive my tresspasses, but I'm now in a position where I really want to move this forward too. Each cloud provider has invested massively in creating SDKs since this package was created. I suggest we reboot the project completely, start from scratch and build a wrapper library for all of the officially supported APIs for all of the major cloud providers. In doing this, we modernise the project and create a simple abstraction API.

**Comment by dantman** [comment]
@walkerandco I agree. Besides rebooting perhaps we should also consider actively culling things.

I would hazard a guess that almost all of the real desire for cross-platform cloud APIs is for a cross-platform cloud files API. Just about everything else feels like something where things are used less often and there is lest cost to just using the official SDK and porting. We can start with just storage and only add other things if there are enough people with a strong desire for it. I believe pkgcloud maintenance was likely significantly burdened by trying to maintain too many things few people need.

Likewise we should also build it monorepo style and place each SDK provider in a separate package. e.g. `@cloud-apis/core`, `@cloud-apis/provider-aws`, `@cloud-apis/google`, etc. This would slim down the number of unnecessary providers SDKs installed. It's also really annoying to have an upstream vulnerability that exists in a SDK for a cloud service you are not using. This is especially painful when that is in a SDK for a service few people use from a provider that is not bothering to maintain their own SDK, e.g. #675.

**Comment by mohd-akram** [comment]
Any update on this?
